core/src/main/scala/cats/syntax/monadTrans.scala

```diff
@@ -0,0 +1,11 @@
+package cats
+package syntax
+
+trait MonadTransSyntax {
+  implicit final def catsSyntaxMonadTrans[F[_], A](fa: F[A]): MonadTransOps[F, A] = new MonadTransOps(fa)
+}
+
+final class MonadTransOps[F[_], A](val fa: F[A]) extends AnyVal {
+  def liftT[MT[_[_], _]](implicit F: Monad[F], MT: MonadTrans[MT]): MT[F, A] =
```

Comment:
I pointed out because codecov reported test miss here. Now I see that `SyntaxTest` is only a compilation check. I think I prefer a doctest in the type class which actually run the code (thus coverage) and provide a concrete example there. But we can certainly address that in a different PR.